cache/caffeine-resteasy/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+io.quarkus.ts.cache.caffeine.restclient.RestInterface/mp-rest/url=http://localhost:${quarkus.http.port}
```

Comment:
Not needed to do it, but maybe we could add here the duration such as : `[quarkus.cache.caffeine.expire-after-write](https://quarkus.io/guides/cache#quarkus-cache_quarkus-cache-caffeine-expire-after-write)` ?

Reply:
Tbh I tried now I didn't make it work. So I'll create small reproducer later and report it to Quarkus. Looked through all TS and we not testing this.

Reply:
It's work as it suppose to, I just tangled into it when creating it